lib/lamby/templates/http.rb

```diff
@@ -10,7 +10,7 @@
 
 say 'Adding to .gitignore...'
 FileUtils.touch app_file('.gitignore')
-append_to_file app_file('.gitignore'), <<-GITIGNORE.strip_heredoc
+append_to_file app_file('.gitignore'), <<~GITIGNORE
```

Comment:
I'm OK with this but I did want to call out it leaves the spaces before it. Not a super big deal, we could also pull that indent to the left.

Reply:
Hmm I'm sure I tested that by switching all the heredocs in the implementation to squiggly heredocs and leaving the `strip_heredoc` in tests.

For all the test cases, the output strings were equivalent.

The [squiggly heredoc](https://ruby-doc.org/core-2.5.0/doc/syntax/literals_rdoc.html#label-Here+Documents) should remove leading whitespace equivalent to the least indented line.

But maybe I'm missing something!

Reply:
Nope, maybe just me learning! I see it now. Thanks!!!